### ISSUE TITLE
Image colors getting inverted by leptonica converter

### ISSUE THREAD

**Issue by nehabagalkot** [bytedeco/javacv#1136]
I am using 
`
LeptonicaFrameConverter c = new LeptonicaFrameConverter();
Frame f = c.convert(src);
BufferedImage img = Java2DFrameUtils.toBufferedImage(f);
`
But the output image has inverted colors, ie 1->0 and vice versa.
using it for .tiff format. How can I resole this?
TIA.

**Comment by saudet** [comment]
We can easily fix that with some simple post-processing, but could you please attach a sample TIFF file that does that here so I can have a look? Thanks!

**Comment by nehabagalkot** [comment]
[New folder.zip](https://github.com/bytedeco/javacv/files/2835368/New.folder.zip)

Refer attachment for both input and output

**Comment by nehabagalkot** [comment]
[New folder.zip](https://github.com/bytedeco/javacv/files/2835376/New.folder.zip)
Attached zip with input and output image.

**Comment by nehabagalkot** [comment]
Its not only for 1-bit image, I tried it for color image, it inverts colors for it as well.
![color_input](https://user-images.githubusercontent.com/13379271/52333317-ef169500-2a22-11e9-864a-5737a5471edd.jpg)
![Uploading color_output.jpg…]()

**Comment by saudet** [comment]
Could you also provide a small but complete code snippet to reproduce this issue?

**Comment by nehabagalkot** [comment]
import java.awt.image.BufferedImage;
import java.io.File;
import javax.imageio.ImageIO;

import org.bytedeco.javacpp.lept;
import org.bytedeco.javacpp.lept.PIX;
import org.bytedeco.javacv.Frame;
import org.bytedeco.javacv.Java2DFrameUtils;
import org.bytedeco.javacv.LeptonicaFrameConverter;
`
public class POCByteArray {

	public static void main(String[] args) throws Exception {
		PIX src = lept.pixRead("RotationTest.tiff");

		LeptonicaFrameConverter c = new LeptonicaFrameConverter();
		Frame f = c.convert(src);
		BufferedImage img = Java2DFrameUtils.toBufferedImage(f);
		ImageIO.write(img, "jpg", new File("RotationTestPIXbyte.jpg"));
		
		
	}
	
}`

**Comment by saudet** [comment]
Right, Leptonica loads all color images with 4 channels, but the order of the color channels doesn't necessarily map well with OpenCV or Java2D, see issue #1075.
